cloud/linode/loadbalancers_test.go

```diff
@@ -285,6 +290,37 @@ func testCreateNodeBalancer(t *testing.T, client *linodego.Client, _ *fakeAPI) {
 	}
 
 	defer func() { _ = lb.EnsureLoadBalancerDeleted(context.TODO(), "linodelb", svc) }()
+	return nil
+}
+
+func testCreateNodeBalancerWithOutFirewall(t *testing.T, client *linodego.Client, f *fakeAPI) {
+	err := testCreateNodeBalancer(t, client, f, nil)
+	if !reflect.DeepEqual(err, nil) {
+		t.Error("unexpected error")
```

Comment:
Can you maybe squash these three lines (here and in the other two test cases) into a single line, something like `t.Fatalf("expected a nil error, got %v" err)`?